AmethystAPI/src/minecraft/src/common/world/inventory/FillingContainer.cpp

```diff
@@ -0,0 +1,44 @@
+﻿#include "FillingContainer.hpp"
+
+#include "minecraft/src/common/world/item/ItemStack.hpp"
+
+extern "C" void* FillingContainer_vtable;
+
+FillingContainer::FillingContainer(Player* player, int size, ContainerType type) : Container(type),
+    mPlayer(player)
+{
+    //workaround to place the original game vtable on the class instance even in a implemented constructor
+    *reinterpret_cast<void**>(this) = FillingContainer_vtable;
```

Comment:
Why is this needed? If you're planning to construct from this, doing asm definitions for this should be good?

Reply:
forgot to remove xD